src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -65,7 +65,7 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         $request->setMethod('GET');
 
         try {
-            $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, true);
+            $response = $event->getKernel()->handle($request, HttpKernelInterface::SUB_REQUEST, false);
```

Comment:
Then it should be confirmed. To me, it looks correct: we are dealing with an exception. If an exception is triggered while rendering the page for our original exception, we don't want to trigger a sub-sub-request to generate (recursively) yet an other exception page for the exception page in the exception page :)
We prefer do just catch the second exception and deal with it like. That's the reason why we have a catch block to me.